TemplateFiles/DeploymentFiles/AzTSDeploymentTemplate.json

```diff
@@ -288,6 +288,14 @@
     "MetadataAggregatorFunctionTimeout": {
       "type": "string",
       "defaultValue": "00:09:00"
+    },
+    "AADClientAppDetailsInstance": {
+      "type": "string",
+      "defaultValue": "https://login.microsoftonline.com/"
+    },
+    "AzureEnvironmentPortalURI": {
+      "type": "string",
+      "defaultValue": "https://portal.azure.com/"
```

Comment:
how we are changing it per environment?

Reply:
This is the default value for the parameter in template and the actual value we are passing from AzTsSetup script as per environemnt like below:  $TemplateParameters.Add("AzureEnvironmentPortalURI", $AzureEnvironmentPortalURI.$AzureEnvironmentName).

I have tested the setup by removing the default value from template.